src/components/tabbed-panels/tabbed-panels.js

```diff
@@ -1,5 +1,24 @@
-(function ($) {
+import { googleAnalytics } from '../../../src/js/googleAnalytics';
```

Comment:
@mlsamuelson This file reference doesn't work. I'm not sure if it is referring to the Google Analytics that is housed in the asu-brand module somewhere, or if this is different. Should I copy the file that is referenced in unity and place it in this theme?

Reply:
Oh. It looks like maybe the file was renamed in WS2 to be "datalayer.js," but the two files are different. Not sure what to do there.

Reply:
@tbutterf That path is digging out from the CSS aggregation folder!

Reply:
@mlsamuelson @tbutterf making a note that I created the datalayer.js file to handle the data layer attribute additions to the twig files in renovation.